Repository/PhpcrOdmRepository.php

```diff
@@ -116,4 +120,37 @@ protected function buildCollection(array $documents)
 
         return $collection;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function removeNodes(array $documents)
+    {
+        foreach ($documents as $document) {
+            $this->getManager()->remove($document);
+        }
+
+        $this->getManager()->flush();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function moveNodes(array $documents, $sourceQuery, $targetPath)
+    {
+        $this->doMoveNodes($documents, $sourceQuery, $targetPath);
+        $this->getManager()->flush();
+    }
+
+    private function doMoveNodes(array $documents, $sourceQuery, $targetPath)
+    {
+        if (false === $this->isGlobbed($sourceQuery)) {
```

Comment:
In Symfony CS, this would be written as just `if (!$this->isGlobbed($sourceQuery)) {` I find that more readable, but I've no hard preference.